src/Components/Payment/__tests__/PaymentForm.test.tsx

```diff
@@ -0,0 +1,280 @@
+import { Button, Checkbox, CheckboxProps, Sans } from "@artsy/palette"
+import { CreditCardInput } from "Apps/Order/Components/CreditCardInput"
+import { mount } from "enzyme"
+import React from "react"
+
+import { creatingCreditCardFailed } from "Apps/Order/Routes/__fixtures__/MutationResults"
+
+jest.mock("react-relay", () => ({
+  commitMutation: jest.fn(),
+  createFragmentContainer: component => component,
+}))
+
+jest.mock("react-stripe-elements", () => ({
+  CardElement: props => <div {...props} />,
+  injectStripe: args => args,
+}))
+
+import { commitMutation, RelayProp } from "react-relay"
+import {
+  Address,
+  AddressForm,
+} from "../../../Apps/Order/Components/AddressForm"
```

Comment:
Now that we have type-checking in on tests all of these relative paths throughout test can be absolute